target/linux/ramips/mt7620/base-files/etc/board.d/02_network

```diff
@@ -186,6 +186,10 @@ ramips_setup_interfaces()
 		ucidef_add_switch "switch1" \
 			"0:lan" "1:lan" "2:lan" "3:lan" "4:wan" "6@eth0"
 		;;
+	netcore,nw5212)
+		ucidef_add_switch "switch0" \
+			"0:lan:3" "1:lan:2" "2:lan:1" "3:lan:0" "4:wan" "6@eth0"
```

Comment:
I'd prefer to start the LuCI number at 1, so
`"0:lan:4" "1:lan:3" "2:lan:2" "3:lan:1" "4:wan" "6@eth0"`

Please also check here whether this can be merged with other devices.